presto-sqlserver/src/test/java/io/prestosql/plugin/sqlserver/TestSqlServerIntegrationSmokeTest.java

```diff
@@ -317,6 +323,36 @@ public void testNativeMultipleInClauses()
         sqlServer.execute("SELECT count(*) FROM dbo.orders WHERE " + longInClauses);
     }
 
+    @DataProvider
+    public Object[][] dataCompression()
+    {
+        return new Object[][] {
+                {NONE},
+                {ROW},
+                {PAGE}
+        };
+    }
+
+    @Test(dataProvider = "dataCompression")
+    public void testCreateWithDataCompression(DataCompression dataCompression)
+    {
+        String tableName = "test_create_with_compression_" + randomTableSuffix();
+        String createQuery = format("CREATE TABLE sqlserver.dbo.%s (\n" +
+                        "   a bigint,\n" +
+                        "   b bigint\n" +
+                        ")\n" +
+                        "WITH (\n" +
+                        "   data_compression = '%s'\n" +
+                        ")",
+                tableName,
+                dataCompression);
+        assertUpdate(createQuery);
+
+        assertEquals(getQueryRunner().execute("SHOW CREATE TABLE " + tableName).getOnlyValue(), createQuery);
```

Comment:
you might want to replace multiple subsequent white characters in to a single space, then value of `createQuery` would be easier to maintain.